mtags-java/src/main/scala/scala/meta/internal/pc/JavaMetalsGlobal.scala

```diff
@@ -19,14 +22,21 @@ class JavaMetalsGlobal(
 
   private val COMPILER: JavaCompiler = ToolProvider.getSystemJavaCompiler()
 
+  private val noopDiagnosticListener = new DiagnosticListener[JavaFileObject] {
+
+    // ignore errors since presentation compiler will have a lot of transient ones
+    override def report(diagnostic: Diagnostic[_ <: JavaFileObject]): Unit = ()
+
+  }
+
   def compilationTask(sourceCode: String, uri: URI): JavacTask = {
     val javaFileObject = SourceJavaFileObject.make(sourceCode, uri)
 
     COMPILER
       .getTask(
         null,
         null,
-        null,
+        noopDiagnosticListener,
```

Comment:
Did it report the errors somewhere when it was a null?

Reply:
It reported it to stderr, co would end up in Metals output, which is not really useful